### DISCUSSION TITLE
Problems renewing my educational teachers license - anyone else experiencing the same?

### DISCUSSION THREAD

**Discussion by astyve** [community/community#53416] [GitHub Education]
Hi,

When trying to renew my educational license (I received e-mails from GitHub that I had to renew the license within 15th April), I end up on a page as shown below: 

<img width="821" alt="Skjermbilde 2023-04-21 kl  13 45 11" src="https://user-images.githubusercontent.com/11460006/233655274-ffea36a2-73d2-4b1b-a49a-91df5cfceeac.png">

If I click the "Request a discount", I end on this page:

<img width="1249" alt="Skjermbilde 2023-04-21 kl  16 00 32" src="https://user-images.githubusercontent.com/11460006/233655672-e6dbc4c8-7ec5-4e5d-81dc-33d6a9a8117b.png">

..and if I click on the "Get teacher benefit", I end up on the previous "404"-page - going in circles.

I've filed a ticket to GitHub support 18 days (!!!) ago, but still no answer, so hence this desperate message in this forum to see if anyone else have experienced similar problems.

Kind Regards,
Arne

**Comment by juanpflores** [comment]
Hi @astyve,
Thanks for reaching out. Sorry to hear that you've been waiting for long. I've reviewed your teacher application and we need you to submit a proof of academic status for us to validate your request. You can find all the details on how to upload your proof [here](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-teachers/apply-to-github-global-campus-as-a-teacher).

**Reply by astyve** [reply]
Hi @juanpflores,

Thanks for at least answering here. I've now got 4 tickets awaiting response from GitHub, the oldest one is 22 DAYS OLD!!

The problem is that I get a 404-page when trying to validate my academic status, and your system is going in circles (as I have tried to explain in two of my submitted tickets). I also note that there are several postings in the community too lately from other users having problem with their academic/educational validation.

Here is the circle-of-pain: I start from here:

<img width="1249" alt="Skjermbilde 2023-04-21 kl  16 00 32" src="https://user-images.githubusercontent.com/11460006/234559703-3942068f-9ce8-4d14-bd36-4a86421d9494.png">

Clicking the "Get teacher benefits"-button, leads me to this page:

<img width="821" alt="Skjermbilde 2023-04-21 kl  13 45 11" src="https://user-images.githubusercontent.com/11460006/234559735-f9fc4cd4-6f66-4841-89a4-e361c62ffff6.png">

Clicking on the "Request discount, returns me to this page:

<img width="1249" alt="Skjermbilde 2023-04-21 kl  16 00 32" src="https://user-images.githubusercontent.com/11460006/234559986-e7b59cdc-82b9-432f-b362-ee57ea8cbe9d.png">

... and the crazy circle is complete!!!

PLEASE can you have someone following up the 22 day old tickets, and fix the 404-circle??

BR,
Arne Styve

**Comment by astyve** [comment]
Hi @juanpflores,

Thanks for at least answering here. I've now got 4 tickets awaiting response from GitHub, the oldest one is 22 DAYS OLD!!

The problem is that I get a 404-page when trying to validate my academic status, and your system is going in circles (as I have tried to explain in two of my submitted tickets). I also note that there are several postings in the community too lately from other users having problem with their academic/educational validation.

Here is the circle-of-pain: I start from here:

<img width="1249" alt="Skjermbilde 2023-04-21 kl  16 00 32" src="https://user-images.githubusercontent.com/11460006/234559703-3942068f-9ce8-4d14-bd36-4a86421d9494.png">

Clicking the "Get teacher benefits"-button, leads me to this page:

<img width="821" alt="Skjermbilde 2023-04-21 kl  13 45 11" src="https://user-images.githubusercontent.com/11460006/234559735-f9fc4cd4-6f66-4841-89a4-e361c62ffff6.png">

Clicking on the "Request discount, returns me to this page:

<img width="1249" alt="Skjermbilde 2023-04-21 kl  16 00 32" src="https://user-images.githubusercontent.com/11460006/234559986-e7b59cdc-82b9-432f-b362-ee57ea8cbe9d.png">

... and the crazy circle is complete!!!

PLEASE can you have someone following up the 22 day old tickets, and fix the 404-circle??

BR,
Arne Styve

**Comment by samus-aran** [comment]
Hi @astyve 👋 

Thank you for bringing this to our attention. We've passed this along to our internal team to investigate and when we have any updates we'll update this thread. 👍

**Reply by astyve** [reply]
Problem has still not been resolved!!!

**Comment by astyve** [comment]
Thanks. Will you also have someone to attend all my tickets to?

Fra: Ashley Nicolson ***@***.***>
Dato: onsdag, 26. april 2023 kl. 15:01
Til: community/community ***@***.***>
Kopi: Arne Styve ***@***.***>, Mention ***@***.***>
Emne: Re: [community/community] Problems renewing my educational teachers license - anyone else experiencing the same? (Discussion #53416)

Hi @astyve<https://github.com/astyve> 👋

Thank you for bringing this to our attention. We've passed this along to our internal team to investigate and when we have any updates we'll update this thread. 👍

**Reply by samus-aran** [reply]
We'll attend to the tickets related to your current issue, yes 👍

**Reply by astyve** [reply]
Still no response to my tickets....

**Reply by astyve** [reply]
Problem still not solved!!
